internal/tenant/tenant.go

```diff
@@ -62,3 +63,38 @@ func (t tenantService) GetTenantInfoByNamespace(apiURL string, ns string) (InfoL
 
 	return tenantInfo, nil
 }
+
+// returns true if the cluster the ns is on has reached maximum capacity
+func (t tenantService) HasReachedMaxCapacity(apiURL, ns string) (bool, error) {
+
+	ti, err := t.GetTenantInfoByNamespace(apiURL, ns)
+	if err != nil {
+		return true, err
+	}
+
+	if len(ti.Errors) != 0 {
+		firstError := ti.Errors[0]
+		err = fmt.Errorf("%s - %s", firstError.Code, firstError.Detail)
```

Comment:
should all the errors not be concatenated instead of returning just the first one?

Reply:
It is really simple logic, why not just do it in this PR?

Reply:
1. this gets the point across to the caller
2.  appending all errors isn't the right thing to do rather, all errors must be output as array of error
e.g. json would now be 


```json
{
  "errors": [
     {"error": "description .."},
     {"error": "description .."},
     {"error": "description .."},
     {"error": "description .."},
  ]
}
```

instead of the current 
```json
 {"error": "description .."},
```

Reply:
Ok then if there is a lot of work to be done, let's do it in another PR but I encourage to do it ASAP or at least open an issue about it, because it is the typical thing that is never done and when there are problems we wish to have all info

Reply:
ack - raised https://github.com/fabric8-services/fabric8-jenkins-idler/issues/231